platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapGestureDetector.java

```diff
@@ -553,7 +566,7 @@ public boolean onRotate(RotateGestureDetector detector) {
       // If rotate is large enough ignore a tap
       // Also is zoom already started, don't rotate
       totalAngle += detector.getRotationDegreesDelta();
-      if (!zoomStarted && ((totalAngle > 20.0f) || (totalAngle < -20.0f))) {
```

Comment:
@Guardiola31337 the removal of zoomstarted indicates that both scale as rotation can occur at the same time. I tried reimplementing to make it work but introduced too many changes. I'm punting on this for now, will write a follow up ticket.